src/main/java/frc/robot/valuetuner/NetworkTableConstant.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.valuetuner;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+/**
+ * The class is used to update the value of the web constant through the networktables.
+ */
+public class NetworkTableConstant implements WebConstant {
+    private static final NetworkTable BASE_TABLE = NetworkTableInstance.getDefault().getTable("value-tuner");
+
+    private final NetworkTableEntry constant;
+    private final double defaultValue;
+
+    NetworkTableConstant(String subsystem, String key, double defaultValue) {
```

Comment:
"subsystem" is too specific, you might want to have subtables for things that aren't subsystems.